aws/data_source_aws_secretsmanager_secret.go

```diff
@@ -104,6 +110,23 @@ func dataSourceAwsSecretsManagerSecretRead(d *schema.ResourceData, meta interfac
 	d.Set("rotation_enabled", output.RotationEnabled)
 	d.Set("rotation_lambda_arn", output.RotationLambdaARN)
 
+	pIn := &secretsmanager.GetResourcePolicyInput{
+		SecretId: aws.String(d.Id()),
+	}
+	log.Printf("[DEBUG] Reading Secrets Manager Secret policy: %s", pIn)
+	pOut, err := conn.GetResourcePolicy(pIn)
+	if err != nil {
+		return fmt.Errorf("error reading Secrets Manager Secret policy: %s", err)
+	}
+
```

Comment:
To prevent errors on Terraform 0.11 and below when referencing the `policy` attribute in modules that may reference secrets with and without policies, we should call `d.Set("policy", "")` before the conditional here. 👍